### DISCUSSION TITLE
Mac OS: DMG install script

### DISCUSSION THREAD

**Discussion by szh** [pyinstaller/pyinstaller#7036] [Help] [answered]
Using `pyinstaller --onefile` on Mac OS to create a .dmg file, is there a way to run a custom script when the app is copied to the Applications folder?
The use case is that I want to add the directory to the users PATH variable or symlink the binary to a location in the PATH when the app is installed so it can be called from a terminal.

**Comment by rokm** [comment, marked as answer]
`pyinstaller --onefile` generates a POSIX executable, not a .dmg file. So anything related to .dmg and installation from it (such as the functionality you describe) is outside the scope of PyInstaller.
